static/js/lib/api.js

```diff
@@ -99,6 +99,10 @@ export async function getPrograms(): Promise<AvailablePrograms> {
   }
 }
 
+export function getProgramLearners(programId: number): Promise<ProgramLearners> {
+  return fetchJSONWithCSRF(`/api/v0/programlearners/${programId}/`, {}, true);
```

Comment:
the `fetchJSONWithCSRF` function no longer takes the `loginOnError` argument that it used to. if you want to have the `loginOnError` functionality (redirecting after a 401) you'll need to do something like what we do in the `getPrograms` function right above.